lib/openxml/contains_properties.rb

```diff
@@ -0,0 +1,24 @@
+require "openxml/has_properties"
+
+module OpenXml
+  module ContainsProperties
+
+    def self.included(base)
+      base.class_eval do
+        include HasProperties
+        include InstanceMethods
+      end
+    end
+
+    module InstanceMethods
+
+      def property_xml(xml)
+        props = active_properties
+        return unless render_properties? props
+        props.each { |prop| prop.to_xml(xml) }
+      end
+
+    end
+
+  end
+end
```

Comment:
Instead of the double-include inside of `class_eval` (+1 for the technique, though), does this work?

```ruby
module OpenXml
  module ContainsProperties
    include HasProperties

    def property_xml(xml)
      # ...
    end

  end
end
```

Reply:
I _think_ just `include HasProperties` here will end up adding `HasProperties::ClassMethods` to `ContainsProperties` instead of to the target class when it `include ContainsProperties`